redfish-core/lib/account_service.hpp

```diff
@@ -997,6 +997,23 @@ inline void
     });
 }
 
+inline void setPropertyAllowUnauthACFUpload(
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp, bool allow)
+{
+    sdbusplus::asio::setProperty(
+        *crow::connections::systemBus, "xyz.openbmc_project.Settings",
+        "/xyz/openbmc_project/ibmacf/allow_unauth_upload",
+        "xyz.openbmc_project.Object.Enable", "Enabled", allow,
+        [asyncResp](const boost::system::error_code& ec) {
+        if (ec)
+        {
+            BMCWEB_LOG_ERROR("D-Bus responses error: {}", ec);
```

Comment:
Should this be ec.value?

(Upstream when I asked this Ed told me ec is okay in BMCWEB_LOG_* context since the format handles it. But not in comparison context.

But seems like in 1110 we have been trying to use ec.value for the message context.)

Reply:
Right, I missed those.